rest-service/manager_rest/storage/management_models.py

```diff
@@ -163,24 +163,16 @@ def groups(cls):
 
     @declared_attr
     def tenants(cls):
-        secondary_table = db.Table(
-            'users_tenants',
+        table_names = ['users', 'tenants', 'roles']
+        columns = [
             db.Column(
-                'user_id',
+                '{0}_id'.format(table_name[:-1]),
```

Comment:
If we're doing this, why not just do `column_names = ['user', 'tenant', 'role']`?

Reply:
The table name is also needed to generate the foreign key, so, despite I don't like much this, the other options are:
- generate the foreign key names from the column_names (similar to this change)
- have two different hardcoded lists (a little bit of duplication)

For now, I think I'm going to stick to the current solution since it's similar to the `many_to_many_relationship` implementation and it will make easier a future refactoring if needed.